db/bpfunc.c

```diff
@@ -520,3 +526,12 @@ static int exec_rowlocks_enable(void *tran, bpfunc_t *func, struct errstat *err)
     return rc;
 }
 
+static int exec_delete_from_sc_history(void *tran, bpfunc_t *func,
+                                       struct errstat *err)
+{
+    BpfuncDeleteFromScHistory *tblseed = func->arg->tblseed;
+    int rc = bdb_del_schema_change_history(NULL, tblseed->tablename, tblseed->seed);
```

Comment:
Why aren't you using the transaction for bdb_del_schema_change_history here?

Reply:
Good catch. Fixing.